src/labels/CustomLabels.labels

```diff
@@ -1945,6 +1945,29 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>bdiAccountCustomIdError</shortDescription>
         <value>The Account Custom Unique ID setting field {0} must have matching fields in the NPSP Data Import object whose API Names are {1} and {2}.</value>
     </labels>
+    <labels>
+        <fullName>bdiAdditionalObjChildNotUpdated</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>bdiAdditionalObjChildNotUpdated</shortDescription>
+        <value>Unable to update the child record's lookup field with new parent record's ID. It was unable to update because of this system error:</value>
```

Comment:
The voice of the second sentence appears to me to be wrong. Shouldn't it read "The record could not be updated because of...."?

There's also a missing article "the" in the first sentence.

Reply:
Thanks for the catch, @davidmreed. I also took out the word "new" because I think it could have been a parent record's ID that was found instead of recently created. @psadie please confirm.
```suggestion
        <value>Unable to update the child record's lookup field with the parent record's ID. The record could not be updated because of this system error:</value>
```